java/opendcs/src/main/java/decodes/cwms/algo/ResEvapAlgo.java

```diff
@@ -303,13 +308,15 @@ public void beforeAllTimeSlices() throws DbCompException
         siteDAO = tsdb.makeSiteDAO();
         timeSeriesDAO = tsdb.makeTimeSeriesDAO();
         crd = new CwmsRatingDao((CwmsTimeSeriesDb) tsdb);
+        CwmsLocationLevelDAO locLevDAO = null;
```

Comment:
Future work, you should be able to call OpenDcsDatabase::getDao here... however, clearly that never gets passed into the algorithm code.